platform/ext/target/nordic_nrf/common/plat_test.c

```diff
@@ -14,14 +14,18 @@
  * limitations under the License.
  */
 
+#include <string.h>
 #include "tfm_plat_test.h"
 #include <stdint.h>
 #include <stdbool.h>
 #include <hal/nrf_gpio.h>
 #include <hal/nrf_timer.h>
+#include <helpers/nrfx_reset_reason.h>
```

Comment:
This file needs to be added. It is not currently present in `common/nrfx/`.

Reply:
Added. Along with nrf_power.h and nrf_reset.h